adr/00-packages.md

```diff
@@ -0,0 +1,44 @@
+# Architecture Decision Record: Package structure
+
+## Context
+
+We need to specify structure for the application. 
+It will involve choosing how we will divide code into packages and modules. 
+In its current state, application resembles an MVC based architecture. 
+We would like to make a clear decision, what application structure should be going forward.
+
+## Decision
+
+We will use onion architecture. We will separate application into layers. Layers will be represented by packages.
```

Comment:
**suggestion:** I'm not sure if onion is something we want to refer to directly. Maybe let's just propose:
```
We will divide the application into layers represented by packages.
```